lib/active_model/validations/chef_version_constraint_validator.rb

```diff
@@ -34,8 +34,9 @@ def initialize(options)
       #
       def validate_each(record, attribute, value)
         Chef::VersionConstraint.new(value)
-      rescue Chef::Exceptions::InvalidVersionConstraint, Chef::Exceptions::InvalidCookbookVersion
-        record.errors.add(attribute, options.fetch(:message), value: value)
+      rescue Chef::Exceptions::InvalidVersionConstraint, Chef::Exceptions::InvalidCookbookVersion => e
+        msg = "#{options.fetch(:message)}. #{e.class}: #{e.message}"
```

Comment:
Good question. There should not be any case where `options[:message]` is nil at this point. A fetch-with-fallback is [provided in the `initialize()`](https://github.com/chef/supermarket/blob/robb/add-detail-to-ver-constraint-errors/lib/active_model/validations/chef_version_constraint_validator.rb#L20-L22) for this validator, so a default value is present already at the point this validation method is called.